src/createServer.js

```diff
@@ -4,0 +1,54 @@
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { inputValidation } = require('./convertToCase/inputValidation');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    try {
+      const [path, queryString] = req.url.split('?');
+      const params = new URLSearchParams(queryString);
```

Comment:
The URL parsing logic here is a bit simplistic and might not handle all cases correctly. Consider using a more robust method for parsing URLs, such as the URL API provided by Node.js.